scripts/rf_bios_settings.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/python
+# Copyright Notice:
+# Copyright 2020 DMTF. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/Redfish-Tacklebox/blob/master/LICENSE.md
+
+"""
+Redfish BIOS Settings
+
+File : rf_bios_settings
+
+Brief : This script uses the redfish_utilities module to manager BIOS settings of a system
+"""
+
+import argparse
+import redfish
+import redfish_utilities
+
+# Get the input arguments
+argget = argparse.ArgumentParser( description = "A tool to manager BIOS settings for a system" )
+argget.add_argument( "--user", "-u", type = str, required = True, help = "The user name for authentication" )
+argget.add_argument( "--password", "-p",  type = str, required = True, help = "The password for authentication" )
+argget.add_argument( "--rhost", "-r", type = str, required = True, help = "The address of the Redfish service (with scheme)" )
+argget.add_argument( "--system", "-s", type = str, help = "The ID of the system to manage" )
+argget.add_argument( "--attribute", "-a", type = str, nargs = 2, metavar = ( "name", "value" ),  help = "Sets a BIOS attribute to a new value" )
+args = argget.parse_args()
+
+# Set up the Redfish object
+redfish_obj = redfish.redfish_client( base_url = args.rhost, username = args.user, password = args.password )
+redfish_obj.login( auth = "session" )
+
+try:
+    # Get the BIOS settings
+    current_settings, future_settings = redfish_utilities.get_system_bios( redfish_obj, args.system )
+
+    if args.attribute is not None:
+        # Based on the current settings, determine the appropriate data type for the new setting
```

Comment:
If doing the `append` action mentioned above, then loop through the attributes here to build up the settings.

Reply:
Fixed